x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -453,3 +457,30 @@ export const getUserProfiles = async (
     return acc;
   }, new Map());
 };
+
+export const validateCustomFieldKeysAgainstConfiguration = ({
+  requestCustomFields,
+  configurationCustomFields = [],
+}: {
+  requestCustomFields: CaseRequestCustomFields;
+  configurationCustomFields?: CustomFieldsConfiguration;
+}): string[] => {
+  const invalidCustomFieldKeys: string[] = [];
+
+  requestCustomFields.forEach((customField) => {
+    let validKey = false;
+    configurationCustomFields.every(({ key: keyInConfiguration }) => {
```

Comment:
`every` is usually used as a predicate. For logic, it is better to use `forEach`.

Reply:
I used `every` to finish the loop early if the corresponding key was found in the configuration

Reply:
I just changed to the lodash solution so not relevant anymore 🙌